script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_2_2.txt

```diff
@@ -1,8 +1,8 @@
 <Page1>
-"........." 
+"........."
 ...I already know.
 If I want to return to a normal life as a human being, I should just run away.
-I'm fine with that. For me, that's simply the best choice.
+I'm fine with that. For me, that would be the best choice.
```

Comment:
I stepped out during the conversation about this line so I didn't hear the resolution, but this is the opposite of the nuance that Petrikow asked to be added back in. He wanted it to sound more like it was a "rational" cost/benefit kinda thing and this is less like that. The "simply" was the "rationalization"/"in context". Original is better imo.